fixtures/big-dipper/lp001.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LP001",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["simo"],
+    "createDate": "2019-08-26",
+    "lastModifyDate": "2019-08-26"
+  },
+  "links": {
+    "manual": [
+      "https://www.manualslib.com/manual/1407632/Big-Dipper-Lp001.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=8HeYSPE-Ejk",
+      "https://www.youtube.com/watch?v=EfBSQLccAHE"
+    ]
+  },
+  "physical": {
+    "dimensions": [255, 255, 365],
+    "weight": 2.8,
+    "power": 180,
+    "bulb": {
+      "type": "54× 3W LEDs (12 red, 18 green, 18 blue, 6 white)"
+    }
+  },
+  "availableChannels": {
+    "Intensity": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [1, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        }
+      ]
+    },
+    "Color Macros": {
+      "defaultValue": 11,
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
```

Comment:
Shouldn't this be shutter closed?

Reply:
Watch the first video (https://www.youtube.com/watch?v=8HeYSPE-Ejk&t=110): Only Intensity and Red are set to 100%, all other channels to 0% and the lamp is red. I therefore interpreted "All light off" as "No lights set by this capability" ...

Reply:
Then why is the defaultValue set to 11?